docs/spec.md

```diff
@@ -5215,18 +5229,18 @@ More formally, for all `result_index` in `index_space(results[0])`:
 
 #### Inputs
 
-| Label | Name         | Type                       | Constraints |
-|-------|--------------|----------------------------|-------------|
-| (I1)  | `inputs`     | variadic number of tensors | (C1)        |
-| (I2)  | `dimension`  | constant of type `si64`    | (C4)        |
-| (I3)  | `is_stable`  | constant of type `i1`      |             |
-| (I4)  | `comparator` | function                   | (C5)        |
+| Label | Name         | Type                                                       | Constraints |
+|-------|--------------|------------------------------------------------------------|-------------|
+| (I1)  | `inputs`     | variadic number of tensors or per-tensor quantized tensors | (C1-C7)     |
+| (I2)  | `dimension`  | constant of type `si64`                                    | (C4)        |
+| (I3)  | `is_stable`  | constant of type `i1`                                      |             |
```

Comment:
That is a very good point. Thanks for pointing out. 

I skimmed through all the uses of if `i1` as indices and found that we use some context to disambiguate them from type usages. For example, `for all i1 < i2 from indices(result)`.  

Nevertheless, I agree that we should be careful to (1) always use such notation with a proper context or (2) use a different letter to represent indices (as `i` is reserved for integer), to avoid future ambiguity. 

cc @burmako

Reply:
I agree for the specification it is possible to tell from the context in the places I’ve seen. Just mentioning in case this may cause any ambiguity in other places these terms are likely to be used (eg reference code/other documents). Maybe, like you say, it is useful to have a different standard letter (such as j or k) for indices where there could be any confusion.

Reply:
Point noted.